manual/running.tex

```diff
@@ -1,59 +1,110 @@
 \chapter{Running QMCPACK}
 \label{chap:running}
 
-To run QMCPACK, put one or more input file names on the command line:
-\begin{verbatim}
-qmcpack [command line options] <XML input file(s)>
-\end{verbatim}
+QMCPACK requires at least one xml input file, and is invoked via:
+
+{\texttt{qmcpack [command line options] <XML input file(s)>}}
 
 \section{Command line options}
 \label{sec:commandline}
-Command line options include
+QMCPACK offers several command line options which affect how calculations
+are performed. If the flag is absent, then the corresponding
+option is disabled.
 
 \begin{description}
-\item[-debug]{ Print more information about code internals.  Helpful for understanding what functions are being called for a particular input file.}
+\item[\texttt{-{}-async\_swap}]{ When enabled, this option will launch a new thread
+  to handle the communication between walkers. If it is not enabled,
+  then the communication happens on the current thread. }
+\item[\texttt{-{}-dryrun}]{ Validate the input file without performing the simulation.
+  This is a good way to ensure that QMCPACK will do what you think it will. }
+\item[\texttt{-{}-gpu}]{ Enables use of the GPU. To use this option QMCPACK must have
```

Comment:
The -gpu option has no effect.  We should remove it from the code and the documentation.